src/client/Client.js

```diff
@@ -195,7 +195,9 @@ class Client extends BaseClient {
 
   /**
    * Logs the client in, establishing a websocket connection to Discord.
-   * @param {string} token Token of the account to log in with
+   * <info>If no token is provided and there hasn't been a connection yet, `process.env.DISCORD_TOKEN`
+   * will be used.</info>
```

Comment:
the token is assigned to `process.env.DISCORD_TOKEN` in the client constructor, would it be more accurate to say it uses `this.token`, and have the docs say it uses the environment variable on the Client class?